### PR TITLE
Adding plugin for pluto

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

**Comment by imbsky** [comment]
Thanks @sudermanjr!

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ itself) to look for plugins.
...
 | Packer                        | [Banno/asdf-hashicorp](https://github.com/Banno/asdf-hashicorp)                                   | [![Build Status](https://travis-ci.org/Banno/asdf-hashicorp.svg?branch=master)](https://travis-ci.org/Banno/asdf-hashicorp)                                                             |
 | PHP                           | [asdf-community/asdf-php](https://github.com/asdf-community/asdf-php)                             | [![Build status](https://github.com/asdf-community/asdf-php/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/asdf-community/asdf-php/actions)                     |
+| Pluto                         | [FairwindsOps/pluto](https://github.com/asdf-pluto)                                               | [![GitHub Actions Status](https://github.com/FairwindsOps/asdf-pluto/workflows/build/badge.svg?branch=master)](https://github.com/FairwindsOps/asdf-pluto/actions)                      |
```
Comment (Stratus3D) [inline README.md]: This URL 404s - `https://github.com/asdf-pluto`.

Comment (imbsky) [reply README.md]: Oops...

Comment (imbsky) [reply README.md]: Fixed in latest master

**Pull request merged**